paper-slider.html

```diff
@@ -198,24 +198,22 @@
         margin: 10px;
         width: calc(100% - 20px);
         height: calc(100% - 20px);
-        border-radius: 50%;
         background-color: var(--paper-slider-knob-color, --google-blue-700);
+        border: 1px solid var(--paper-slider-knob-color, --google-blue-700);
+        border-radius: 50%;
 
         -moz-box-sizing: border-box;
         box-sizing: border-box;
 
-        transition-property: margin, height, width, background-color, border;
```

Comment:
It still does layout on certain things (e.g. dragging to the "start" state still transitions `background-color` and `border`). I now transition from `border: 2px` to `1px` when expanding to maintain a similarly-sized border after scale, but at least that only happens at the end (which is better than layout everywhere).